privet/avahi.go

```diff
@@ -4,11 +4,12 @@
 // license that can be found in the LICENSE file or at
 // https://developers.google.com/open-source/licenses/bsd
 
-// +build linux
+// +build linux freebsd
 
 package privet
 
-// #cgo LDFLAGS: -lavahi-client -lavahi-common
+// #cgo CFLAGS: -I%%PREFIX%%/include
```

Comment:
OK, this builds fine on OSX, and I didn't try Linux because I found this solution.

Change to `#cgo freebsd CFLAGS: -I...`